.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,73 @@
+name: Bump versions for extension only
+
+on:
+    push:
+      branches: 
+        - 'master'
+        - '**.x'
+    workflow_dispatch: # promote patch to stable
```

Comment:
Seems the whole script is doing the same thing twice - once for the normal case and once for `workflow_dispatch` - is this correct? If so, would it possibly make sense to just split this into 2 workflows?

Reply:
Yes, it runs for a push to a patch branch and on push to master, so it releases a new Insider extension. When manually triggered, it promotes the patch branch to stable and publishes a new stable extension version. 

To make this less confusing, I can split this this up though.

Reply:
Even reading your sentence 3 times I am not fully sure I got that, so might be worth doing.